### DISCUSSION TITLE
Is there a plan to add Autocomplete component

### DISCUSSION THREAD

**Discussion by amralsaidy** [chakra-ui/chakra-ui#9377] [General] [answered]
### Description

Is there a plan to add Autocomplete component

### Problem Statement/Justification

An essential tool in all projects and available on all other platforms.

### Proposed Solution or API

<Autocomplete/>

**Comment by veloware** [comment]
There's an open PR for the combobox https://github.com/chakra-ui/chakra-ui/pull/9199

**Comment by segunadebayo** [comment, marked as answer]
Use Combobox. That is the autocomplete. https://chakra-ui.com/docs/components/combobox
